### ISSUE TITLE
Multiple choice voting numeration is broken

### ISSUE THREAD

**Issue by anettlinno** [citizenos/citizenos-fe-old#966]
**What is the problem?**
User initiated multiple choice voting with 53 defined options. Options numeration is following the alphabet sequence but is also consisting characters like **[.** and **\.** and **^.** etc. AND alphabet is repeated again from the beginning (a, b, c, etc). Additionally in the results page voting options are presented randomly and not in sequence

![IOF](https://user-images.githubusercontent.com/45175679/156773621-79b37b66-397e-4fab-99b9-6d6c188e50f2.png)

**Why is this a problem?**
Page looks broken and it´s hard to follow voting options as they are melting into the text. 

**Possible solution.**
- Please fix the numeration
- Voting options could be shown one after the other in a list

**Comment by loorm** [comment]
"Page looks broken and it´s hard to follow voting options as they are melting into the text."

That's because you should not have 53 voting options!

I suggest limiting the number of voting options to then number of letters in the Latin alphabet (23). even that's too many options, but so be it.

**Comment by ilmartyrk** [comment]
@loorm I agree that 53 is really really extreme case. I fixed the numeration and displaying of options from being inline to list. This makes it better to follow if option values are long. But I believe that adding limit will not fix any issues.  
1) There is no big performance hit to our system
2) There could be this kind of extreme cases where n+x amount options are needed. Eg. voting to choose representor for a parliament size group of people. Why exclude them from using our platform?
3) At the same time most users will probably use "normal" number of options anyway
